criação da entidade produto/ Produto.cs/ ProdutosQueryHandler.cs

alef4125 opened the pull request.

alef4125 committed:
criação da entidade produto/ Produto.cs/ ProdutosQueryHandler.cs

alef4125 committed:
ajustes nas classes ProdutoDto.cs/ ProdutosQuery.cs/ ProdutosQueryHandler.cs
e criação da IProdutoRepository.cs

alexrochaf commented:
Lembrar de sempre por seu usuário como `assigner` do `pull request`

<img width="332" alt="Captura de Tela 2025-05-08 às 19 35 48" src="https://github.com/user-attachments/assets/ac51b58f-e10d-46e2-9475-a5f875ab06b4" />

alexrochaf commented:
Você está fazendo o merge na `branch` de `dev` mas deveria ser na `branch` de `treinamento` que criamos.

alexrochaf commented:
Você pode ver que teve um `check failed` no seu merge, isso é para garantir que não suba código quebrado, o `build` que você subiu está quebrando, provavelmente pela `interface` que você criou e não implementou.

alef4125 committed:
ajustes nas class

alef4125 committed:
Ajustes nas classes

alef4125 committed:
criação das classes CreateProdutoCommand.cs/ ListProdutoQuery.cs/ ProdutoEndpoints.cs/
ProdutoViewModel.cs todas em andamento

exclusão das classes IProdutoRepository/ ProdutoRepository/ProdutosController/ ProdutosQueryHandlerTests

alef4125 committed:
Ajustes nas classes CadastroDbContext.cs/ CreateProdutoCommand.cs/ ListProdutosQuery.cs/ ProdutoEndpoints.cs/ ProdutoQuery.cs/ ProdutosDto.cs/ ProdutosQueryHandler.cs/ ProdutoViewModel.cs

Criação da classe CreateProdutoCommandHandler.cs/ CompanyProduto.cs

alef4125 committed:
Ajustes nas classes CreateProdutoCommand.cs/ ProdutoQuery.cs/  ProdutoQueryHandler.cs/  ProdutosDto.cs

CreateProdutoCommandHandler.cs ainda em ajustes

alef4125 committed:
Ajustes nas classes CreateProdutoCommandHandler.cs/ CadastroDbContext.cs/ Produto.cs/ ProdutoQuery.cs

alef4125 committed:
criação da classe ProdutoConfiguration.cs e início do desenvolvimento

alef4125 committed:
ajustes de nomenclatura nas classes
e criação da migration AddProducts

alef4125 committed:
Ajuste de nomenclatura nas classes CreateProductCommand.cs/ CreateProductCommandHandler.cs/ ProductQuery.cs/ ProductQueryHandler.cs/ ProductsDto.cs

alef4125 committed:
ajustes de indentação das classes

alef4125 committed:
criação da classe AddProducts.Designer.cs
